src/secrets/quote.txt

```diff
@@ -0,0 +1 @@
+There are no secrets better kept than the secrets everybody guesses.
```

Comment:
This file and directory (inherited from the skeleton) can be deleted, unless you plan on using them for something.

Reply:
@dav3r I have removed that directory as I have no plans for it.